setup-tests.ps1

```diff
@@ -0,0 +1,43 @@
+# Check if we're running on Windows, otherwise return as we have nothing to do on other platforms
+if ($IsWindows -eq $false) {
+  return
+}
```

Comment:
This is still a syntax error in bash because it uses parentheses instead of square brackets.

Reply:
To be clear you don't need this to be syntactically valid in bash, you just need a check in the postinstall script or something.

Reply:
This is a power shell script, not bash

Reply:
I suppose you'll need two checks; if powershell isn't available don't run the script at all and then if it's not windows then don't add the junction.

I thought you were hoping that the condition would happen to be valid in Bash so that you could unconditionally run this but apparently not.

Reply:
Yes, then we should check that the filesystem is ntfs too, as it won't work on others and for the symlinks check that it's ext2/3/4 as that's what it takes

Reply:
The vast majority of people on a non-NTFS system will have the the git behaviour of clconing symlinks intact.

If that lets some people slip through the cracks so be it. I mean I'm not opposed to handling this anyways I suppose but only if you're up for the extra effort.

Reply:
I totally agree.

Let's put this on hold for a little while, I want to explore some alternative ideas, and if they don't pan out we'll put this together as you've expressed.

One question though, as I'm new to this. Does anything we do with this affect the end usage of this library?

I mean, we're working in the tests folder and intuitively that would mean only for collaborators of this repo. This was my original assumption and why I didn't plan on making this windows/ntfs only step automated, but something a collaborator on windows could run in their shell as a one time thing after cloning.

Reply:
I tried removing `/tests/node_modules` and added
```json
dependencies: {
...
"@types/fvtt-types": "file:."
...
}
```
to `package.json`

I can then run `npm run tests` as before, without any symlinks.

Reply:
My testing suggests this works

Reply:
I'll make a clean pull request with the new solution when I'm at the computer